src/settings_internal.js

```diff
@@ -238,6 +238,11 @@ var HAS_MAIN = false;
 // Set to true if we are linking as C++ and including C++ stdlibs
 var LINK_AS_CXX = false;
 
+// Set when closure compiler may be run: Either emcc will run it, or the user
+// might run it after emcc. Either way, some JS changes and annotations must be
+// emitted in that case for closure compiler.
+var MAYBE_CLOSURE_COMPILER = false;
```

Comment:
How about calling this something like "CLOSURE_COMPATIBLE"?    I'm not sure why we need MAYBE in the name.

I guess `--minify=0` isn't quite the same as `--closure-compatible` but we could always add the latter later if we want.. but the internal name I think doesn't need maybe in it does it?

Reply:
We have other `MAYBE_*` uses in the codebase, like `MAYBE_WASM2JS`, so this is consistent with that.

I think it's unambiguous what "maybe" means? But changing them all to "compatible" makes equal sense to me I guess. That is, as the comment says this setting is true when closure might be run (which is separate from using `minify=0` to determine that - we currently use that as an indicator, but we might change that later. if we do, this setting and uses of it will still make sense with no rewriting except for the code that sets the setting or not)

Reply:
Since its internal I don't really mind.   lgtm either way